src/tests/universal-nrp-e2e/StartTests.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# StartTests.sh
+# Description: This Microsoft internal only script is used to download the latest Azure Policy packages from internal pipelines
+#              and run the tests on the specified test data provided.
+# Usage: ./StartTests.sh [-r run-id] [-m vm-memory-mb] [-j max-concurrent-jobs]
+#        -r run-id: Specify the run-id of the pipeline run to download the packages from (Default: latest-succeeded-run-id)
+#        -m vm-memory-mb: Specify the memory in MB to be used for the VMs (Default: 512)
+#        -j max-concurrent-jobs: Specify the maximum number of concurrent jobs to run the tests (Default: 5)
+
+test_data='[
+    {"imageFile": "centos-7.qcow2", "policyPackage": "AzureLinuxBaseline.zip", "resourceCount": 168},
+    {"imageFile": "centos-7.qcow2", "policyPackage": "LinuxSshServerSecurityBaseline.zip", "resourceCount": 20},
+    {"imageFile": "centos-8.qcow2", "policyPackage": "AzureLinuxBaseline.zip", "resourceCount": 168},
+    {"imageFile": "centos-8.qcow2", "policyPackage": "LinuxSshServerSecurityBaseline.zip", "resourceCount": 20},
+    {"imageFile": "debian-10.qcow2", "policyPackage": "AzureLinuxBaseline.zip", "resourceCount": 168},
+    {"imageFile": "debian-10.qcow2", "policyPackage": "LinuxSshServerSecurityBaseline.zip", "resourceCount": 20},
+    {"imageFile": "oraclelinux-7.qcow2", "policyPackage": "AzureLinuxBaseline.zip", "resourceCount": 168},
+    {"imageFile": "oraclelinux-7.qcow2", "policyPackage": "LinuxSshServerSecurityBaseline.zip", "resourceCount": 20},
+    {"imageFile": "rhel-7.qcow2", "policyPackage": "AzureLinuxBaseline.zip", "resourceCount": 168},
+    {"imageFile": "rhel-7.qcow2", "policyPackage": "LinuxSshServerSecurityBaseline.zip", "resourceCount": 20},
+    {"imageFile": "sles-12.qcow2", "policyPackage": "AzureLinuxBaseline.zip", "resourceCount": 168},
+    {"imageFile": "sles-12.qcow2", "policyPackage": "LinuxSshServerSecurityBaseline.zip", "resourceCount": 20},
+    {"imageFile": "ubuntu-16.04.qcow2", "policyPackage": "AzureLinuxBaseline.zip", "resourceCount": 168},
+    {"imageFile": "ubuntu-16.04.qcow2", "policyPackage": "LinuxSshServerSecurityBaseline.zip", "resourceCount": 20},
+    {"imageFile": "ubuntu-18.04.qcow2", "policyPackage": "AzureLinuxBaseline.zip", "resourceCount": 168},
+    {"imageFile": "ubuntu-18.04.qcow2", "policyPackage": "LinuxSshServerSecurityBaseline.zip", "resourceCount": 20}
+]'
+
+subscriptionId="ce58a062-8d06-4da9-a85b-28939beb0119"
+storageAccount="osconfigstorage"
+containerName="diskimages"
+pipelineId=394569
```

Comment:
Is there a way we can get the pipeline run auto-magically? Or maybe just require it as a param instead of hardcoding?

Reply:
We could, but the intent was to just have a script we can just run without any arguments. Its not meant to be reusable by the public, this is an internal-only script.